astropy/coordinates/sky_coordinate.py

```diff
@@ -1003,6 +1007,105 @@ def to_string(self, style='decimal', **kwargs):
 
         return coord_string
 
+    def to_table(self, table=None, column_conflicts='warn',
+                 metadata_conflicts='warn'):
+        """
+        Convert the data in this |SkyCoord| to a |QTable|.
+
+        Any attributes that have the same length as the |SkyCoord| will be
+        columns in the output. All other attributes will be recorded as
+        metadata.
+
+        If ``table`` is a |Table| whose length matches that of the |SkyCoord|
+        then the columns and metadata of the converted |SkyCoord| are merged
+        with it. In case of a column name or metadata conflict data from the
+        |SkyCoord| is preferred.
```

Comment:
```suggestion
        with it. In the case of a metadata or column name conflict, data from the
        |SkyCoord| are preferred.
```

Reply:
I'll wait a couple of days before pushing the changes in case there will be more feedback to incorporate.